unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SignupHUD/SignupHUDController.cs

```diff
@@ -51,6 +51,7 @@ public class SignupHUDController : IHUD
             this.browserBridge = browserBridge;
             this.subscriptionsAPIService = subscriptionsAPIService;
             loadingScreenDataStore.decoupledLoadingHUD.visible.OnChange += OnLoadingScreenAppear;
+            DataStore.i.backpackV2.isWaitingToBeSavedAfterSignUp.OnChange += OnTermsOfServiceAgreedStepAfterSaveBackpack;
```

Comment:
inject the datastore instead of referring to the singleton

Reply:
true! On it!